widgets/nuxeo-uploader-behavior.html

```diff
@@ -310,6 +308,9 @@
               this._uploadProgressUpdated(event.fileIdx, event.progress);
               break;
             case 'uploadCompleted':
+              if (!this.batchId) {
+                this.batchId = event.batchId;
+              }
```

Comment:
I would check the situation when `this.batchId` exists but it is different from `event.batchId`

Reply:
If the batch changes between the uploader. i think it is another problem because already the batchId is computed at the end of batch upload and if we cancel it will mean we cancel the wrong batchId.

I dont't think that the batchId change is because it is computed once at the start of the upload and if it exists it will be not computed for the same instance of uploader [here](https://github.com/nuxeo/nuxeo-js-client/blob/master/lib/upload/batch.js#L100).


I am going to change the things to have more consistency between the behaviour and the underline Uploader.